certified-connectors/DocuWare/README.md

```diff
@@ -0,0 +1,83 @@
+## DocuWare Power Automate Connector
+
```

Comment:
Can you please update readme.md file as per the below template?

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
I've updated the readme to what you're adviced